app/_landing_pages/gateway/topology-hosting-options.yaml

```diff
@@ -0,0 +1,227 @@
+                    Serverless gateways are ideal for developers who want to test or experiment in a pre-production environment.
+
+                    Serverless gateways offer the following benefits:
+
+                    * {{site.konnect_short_name}} manages provisioning and placement
+                    * Can be deployed in under 30 seconds
+                    * Access to Kong Gateway plugins
+          - type: button
+            config:
+              align: left
+              text: Learn more about serverless gateways
+              url: /serverless-gateways/
+  - header:
+      type: h3
+      text: "Supported deployment topologies"
+    columns: 
+      - blocks:
+        - type: card
+          config:
+            title: "{{site.konnect_short_name}} hybrid mode"
+            description: |
+              Learn about {{site.konnect_short_name}} hybrid mode for Kong-hosted Control Planes.
+              
+            icon: /assets/icons/deployment.svg
+            cta:
+              url: "/gateway/hybrid-mode/"
+  - header:
+      type: h2
+      text: "Dedicated Cloud Gateways"
+    columns:
+      - blocks:
+          - type: mermaid
+            config:
+              diagram: |
+                flowchart TD
+                A(Dedicated Cloud Gateway Control Plane)
+                B(Managed Data Plane Node \n Region 1)
+                C(Managed Data Plane Node \n Region 2)
+                D(Self-Managed Data Plane Nodes)
+
+                subgraph id1 [Konnect]
+                A
+                end
+
+                A --auto-scale configuration---> B
+                A --auto-scale configuration---> C
+                A -.- D
+
+                style id1 stroke-dasharray:3,rx:10,ry:10
+                style A stroke:none,fill:#0E44A2,color:#fff
+                style B stroke:none,fill:#0E44A2,color:#fff
+                style C stroke:none,fill:#0E44A2,color:#fff
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    Dedicated Cloud Gateways are Data Plane nodes that are fully-managed by Kong in {{site.konnect_short_name}}.
+                    
+                    You don't need to host any Data Planes, and you maintain control over the size and location of the gateway infrastructure. 
+                    This allows Kong to autoscale your nodes for you and reduces your operational complexity.
+
+                    Benefits of Dedicated Cloud Gateways:
+                    - {{site.konnect_short_name}} handles gateway upgrades for you.
+                    - Choose between public and private networking modes.
+                    - Automatic scaling with Autopilot mode.
+                    - Support for AWS and Azure in multiple [regions](/konnect-geos/).
```

Comment:
```suggestion
                    - Support for AWS, Azure and GCP in multiple [regions](/konnect-geos/).
```

Reply:
Looks like not yet: https://konghq.aha.io/epics/KP-E-581